packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -31,17 +31,42 @@
 					:parameters="parametersNoneSetting"
 					:hideDelete="true"
 					:nodeValues="nodeValues" path="parameters" @valueChanged="valueChanged"
+					:isSupportedByHttpRequestNode="isSupportedByHttpRequestNode"
 				>
+					<n8n-notice
+						v-if="isHttpRequestNodeV2(node) && scopes.length > 0"
+						:content="scopesContent"
+						:truncateAt="scopesContent.indexOf('<br>')"
+						:trailingEllipsis="false"
+						:expandFromContent="true"
+						:expansionTextPattern="/\d+ scopes?/"
```

Comment:
we need to rethink this approach here. component is not generic enough. also does not support i18n... 
What we should implement instead is something like this:
```
<n8n-notice :content="lesscontent" :expandedContent="fullcontent"  />
```
within the component, we can listen to all link clicks
```
methods: {
 onClick(e) {
  if (e.target.tag !== 'a') {
    return;
  }
  if (e.target.dataset.key === 'show-less') {
    this.showFullContent = false;
  }
 else if (e.target.dataset.key === 'expand') {
    this.showFullContent = true;
 }
}
```

and in en.json
```
"fullcontentkey": "whatever ... <a data-key="expand">scopes</a> whatever"
"lesscontentkey": "whatever scopes whatever ... <a data-key="show-less"></a>"
```
this will allow us to reuse this functionality more easily and expand it to support any events Product dream up.

Reply:
Thank you. Between all the back and forth between the feedback and the original component design and Alex's implementation, I was not sure what should still be kept and what could be ignored. Will move to this.